nixos/tests/hibernate.nix

```diff
@@ -36,7 +36,7 @@ import ./make-test.nix (pkgs: {
       $machine->waitForShutdown;
       $machine->start;
       $probe->waitForUnit("network.target");
-      $probe->waitUntilSucceeds("echo test | nc -c machine 4444");
+      $probe->waitUntilSucceeds("echo test | nc -q 0 machine 4444");
```

Comment:
This is the equivalent to `-c` in netcat-openbsd. I couldn't find another way to accomplish closing at `EOF`. I also could not override the `systemPackages` to include netcat-gnu instead of netcat-openbsd.